components/org.wso2.carbon.identity.oauth2.grant.organizationswitch/src/main/java/org/wso2/carbon/identity/oauth2/grant/organizationswitch/OrganizationSwitchGrant.java

```diff
@@ -232,55 +234,31 @@ private Optional<org.wso2.carbon.user.core.common.User> getFederatedUserFromResi
     }
 
     /**
-     * Builds the revocation request and calls the revoke oauth service.
+     * Update the token binding reference with the new token id.
```

Comment:
why do we have to do this?
Won't the binding value get set automatically in the DB?